### DISCUSSION TITLE
Webstorm page routes awareness

### DISCUSSION THREAD

**Discussion by DanieleGBX** [vercel/next.js#12329] [Help]
Is there any way to make webstorm aware of the routes generated by the pages?

Moreso, is it possible to enforce type validation for querystring parameters?

**Comment by ChristianIvicevic** [comment]
> Is there any way to make webstorm aware of the routes generated by the pages?

This discussion is very old, but I was curious whether you can share more information on that. What is it that you would have liked to see supported by WebStorm.
